app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,26 @@
+class InvitationsController < ApplicationController
+  before_action :require_user
+
+  def new
+    @invitation = Invitation.new
+  end
+
+  def create
+    @invitation = Invitation.create(invitation_params.merge!(inviter_id: current_user.id))
+
+    if @invitation.save
+      AppMailer.send_invitation_email(@invitation).deliver
+      flash[:success] = "Your invitation is on it's way!"
+      redirect_to new_invitation_path
+    else
+      flash[:danger] = "Something went wrong, please try again."
```

Comment:
@brandoncc (sorry, it's been a while!) so should I use `flash.now` when I'm rendering and then `flash` when I'm redirecting? 
Tried `flash.now` with both and it does not seem to work if the subsequent action is a redirect.